content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,152 @@
-servicename: /etc/init.d/datadog-agent
-serviceinfoname: /etc/init.d/datadog-agent info
-configdirectory: /etc/dd-agent/
-logdirectory: /var/log/datadog/
-supervisorlog: /var/log/supervisor/datadog-supervisord.log
 aliases:
     - /guides/basic_agent_usage/amazonlinux/
 ---
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent.
+This page outlines the basic functionality of the Datadog Agent for Amazon linux.  
 If you haven't installed the Agent yet, instructions can be found
 [in the Datadog agent integration page](https://app.datadoghq.com/account/settings#agent/aws)
 
-The process to upgrade from the previous version of the agent is to re-run the installation.
+## Commands
 
-## Starting and Stopping the Agent
+The service manager provided by the operating system is responsible for the Agent lifecycle: depending on which one is installed, `sudo service`, `sudo initctl` or `sudo systemctl` should be used to `start`, `stop` or `restart` the Agent.  
+Other functionalities are provided by the Agent binary itself, globally available through the command `datadog-agent`: it's the case for example of the `status` and `flare` commands.
 
-To manually start the Agent:
-```shell
-sudo /etc/init.d/datadog-agent start
-```
+{{% table responsive="true" %}}
+| Agent v5                                  |  Agent v6                          | Notes
+| ----------------------------------------------- | --------------------------------------- | ----------------------------- |
+| `sudo service datadog-agent start`              | `sudo service datadog-agent start`      | Start Agent as a service |
+| `sudo service datadog-agent stop`               | `sudo service datadog-agent stop`       | Stop Agent running as a service |
+| `sudo service datadog-agent restart`            | `sudo service datadog-agent restart`    | Restart Agent running as a service |
+| `sudo service datadog-agent status`             | `sudo service datadog-agent status`     | Status of Agent service |
+| `sudo service datadog-agent info`               | `sudo datadog-agent status`             | Status page of running Agent |
+| `sudo service datadog-agent flare`              | `sudo datadog-agent flare`              | Send flare |
+| `sudo service datadog-agent`                    | `sudo datadog-agent --help`             | Display command usage |
+| `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent -- datadog-agent check <check_name>` | Run a check |
+{{% /table %}}
 
-To stop the Agent:
-```shell
-sudo /etc/init.d/datadog-agent stop
-```
+**NB**: If `service` is not available on your system, use:
 
-To restart the Agent and reload the configuration files:
-```shell
-sudo /etc/init.d/datadog-agent restart
-```
+* on `upstart`-based systems: `sudo start/stop/restart datadog-agent`
+* on `systemd`-based systems: `sudo systemctl start/stop/restart datadog-agent`
+* on `initctl`-based systems: `sudo initctl start/stop/restart datadog-agent`
```

Comment:
`initctl` is the same thing as upstart (actually, `initctl` is the command to communicate with upstart). `start`, `stop` and `restart` are aliases for respectively`initctl start`, `initctl stop`, `initctl restart`.

So I think we should merge this line with the `upstart` one and reword it: `* on upstart-based systems: sudo initctl start/stop/restart datadog-agent` (I think it's better to be explicit and use initctl)

This applies to all the linux docs pages :)

Reply:
👌